oak_tensorflow_service/src/tflite.rs

```diff
@@ -37,7 +37,7 @@ pub unsafe extern "C" fn oak_log_debug(message_ptr: *const core::ffi::c_char, me
 }
 
 // TODO(#3297): Decide the TensorFlow model format to be passed as `model_bytes`.
-#[link(name = "hello_world_app", kind = "static")]
+#[link(name = "core_full_static", kind = "static")]
```

Comment:
Static is already implied in `kind = "static"`

Reply:
`full static` is used to differentiate the library archive that bazel generates given bazel doens't build real static library by default.